src/App.jsx

```diff
@@ -1,10 +1,99 @@
 import React from "react";
 import { render } from "react-dom";
-import Home from "./pages/Home/index.jsx";
-import "./styles.scss";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import MainLayout from "./pages/MainLayout/index.jsx";
+import styles from "./styles.module.scss";
 
 const App = () => {
-  return <Home />;
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <MainLayout
+              title="Personal information"
+              nextPath="/programming_languages"
```

Comment:
Nice. but also better to use `-` 
![image](https://user-images.githubusercontent.com/1394580/211294067-27ff63b5-5c3e-402f-a7af-281be8228c5d.png)
https://blog.restcase.com/5-basic-rest-api-design-guidelines/

Reply:
done